tests/metrics/test_LPIPS.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import torch
+from skimage import data
+from torchmetrics.image.lpip import LearnedPerceptualImagePatchSimilarity
+
+from meddlr.metrics.LPIPS import LPIPS
+
+
+def test_LPIPS_torchmetrics_reproducibility():
+    """Test reproducibility between SSIM implementations in meddlr and torchmetrics."""
+
+    meddlr_metric = LPIPS(mode="rgb", lpips=True)
+
+    preds = torch.randn((5, 3, 256, 256))
+    targets = torch.randn((5, 3, 256, 256))
```

Comment:
also have a test with a stock RGB image from skimage?

Reply:
instead of the random image or in addition to?

Reply:
in addition to - you can probably make more sense of a structured image than an unstructured (i.e. noise) one